apps/api-journeys/src/app/modules/action/action.resolvers.spec.ts

```diff
@@ -1,10 +1,14 @@
 import { Test, TestingModule } from '@nestjs/testing'
-import { RadioOptionBlock } from '../../__generated__/graphql'
+import { Database } from 'arangojs'
+import { mockDeep } from 'jest-mock-extended'
+import { Action, RadioOptionBlock } from '../../__generated__/graphql'
 import { BlockResolvers } from '../block/block.resolvers'
 import { BlockService } from '../block/block.service'
+import { UserJourneyService } from '../userJourney/userJourney.service'
+import { ActionResolver } from './action.resolvers'
 
 describe('ActionResolvers', () => {
-  let resolver: BlockResolvers
+  let resolver: BlockResolvers, actionResolver: ActionResolver
```

Comment:
Can you swap these?
```suggestion
  let resolver: ActionResolver, blockResolver: BlockResolvers
```

Can we also change the name of ActionResolver to ActionResolver**s**